Add gitarro to git tools

MalloZup opened the pull request:
**Thanks for contributing to Awesome Ruby! Please take a look at the [contribution guidelines and quality standard](https://github.com/markets/awesome-ruby/blob/master/CONTRIBUTING.md) first.**

## Project

https://opensuse.github.io/gitarro/

## What is this Ruby project?

gitarro allow to run custom tests on GitHub Pull Requests.
Gitarro use the official Github API, written in Ruby (Octokit)

## What are the main difference between this Ruby project and similar ones?
There is no such a project for ruby at moment.

___


Please help us to maintain this collection by using reactions (:+1:, :-1:, ...) and comments to express your feelings.

markets commented:
Thanks for sharing @MalloZup 👍  

Still too fresh for this list, only 400 downloads according RubyGems... lets keep it open for some time.

MalloZup commented:
hi @markets, i reping you on this one: 
https://rubygems.org/gems/gitarro now is at ~ 5 670 Downloads

And 1 year has passed :smile: 
Personally i don't think that metrics download should be really important.
Anyway gitarro fullfill imho the standards
https://github.com/markets/awesome-ruby/blob/master/CONTRIBUTING.md#quality-standard.

For any infos required feel free to ask thanks

markets commented:
Hi @MalloZup 👋 

Metrics about downloads is quite good indicator in my opinion, it indicates if a project is used or not used by the community. 

I know it shouldn't be the only reason to take a decision, and it is not ... but numbers help a lot to draw a line in these `awesome-*` collections. I'm also taking into account other indicators as: GH stars (popularity), GH reactions/comments (community feedback)... and of course all the other the points in the quality standard section. 

(You can check similar discussions in: #60, #597, #392, #304, #384)

I think I'll introduce some quantitative references in that section (ie. 10k downloads, 30 stars, 2 contributors, 3 upvotes in the PR, 1 year of life, etc.). That will facilitate maintenance and, by consequence, keep the value of this list.

That said, I revisited `gitarro` docs and repo, and I think it probably deserves a place here. So, could you please update the PR according the contrib guides? I'll happy to merge!

MalloZup commented:
@markets thanks :+1: i am really happy about this because i did since the 1st day gitarro to be much opensource as possible. For me is a good achievement to have `gitarro` in this list and i hope that more people will use it and we and i can improve it from other users :+1: 

i will create another pr and link to this one because `unknown repository` , i think i have deleted the repo.. :smile:  sorry for the unconvenience and i will add the PR linked here. :white_flower: :tulip:

MalloZup commented:
@markets  i will close this PR and i have added this one (because  : 

https://github.com/markets/awesome-ruby/pull/848

feel free to ping me for any info. (i have added `gitarro` in Testing extra, feel free to suggest another category)

Thanks and nice weekend :sunflower:

markets commented:
Closed by #848 

Thanks!

MalloZup commented:
@markets  thank you ! :tada: :tada:
